website/sidebars.js

```diff
@@ -1157,10 +1157,9 @@ const sidebarSettings = {
         {
           type: "category",
           label: "For macros",
-          link: { type: "doc", id: "reference/macro-properties" },
+          link: { type: "doc", id: "reference/resource-properties/arguments" },
```

Comment:
This needs to be reverted back to the original:`link: { type: "doc", id: "reference/macro-properties" },